crates/net/network/src/network.rs

```diff
@@ -132,7 +132,7 @@ impl NetworkHandle {
 
     /// Announce a block over devp2p
     ///
-    /// Caution: in PoS this is a noop, since new block propagation will happen over devp2p
+    /// Caution: in PoS this is a noop, since new block propagation will happen over libp2p
```

Comment:
They're still available on devp2p but no longer announced, 
the CL propagates blocks, which happens over libp2p but is not important here.

mind adding this smol nuance?